apps/dashboard/src/components/workflow-editor/steps/email/maily-config.tsx

```diff
@@ -55,5 +61,20 @@ export const getDefaultEditorBlocks = (isCustomEmailBlocksEnabled: boolean): Blo
     ],
   });
 
+  blocks.push({
+    title: 'Components',
+    commands: [
+      {
+        id: 'headers',
+        title: 'Headers',
+        description: 'Header: Logo with cover image',
+        searchTerms: ['header', 'headers'],
+        icon: <EmailHeader className="size-4" />,
+        preview: '/images/email-editor/header-preview.png',
+        commands: [headerLogoWithCoverImage, headerCenteredLogoWithBorder, headerLogoWithText],
+      },
+    ],
+  });
+
```

Comment:
Please move the header in the "Highlights" ([ref](https://linear.app/novu/issue/NV-5484/reorder-slash-menu-blocks)):

```tsx
  if (isCustomEmailBlocksEnabled) {
    blocks.push({
      title: 'Highlights',
      commands: [htmlCodeBlock],
    });
  }
 ```